drivers/hmis_external_apis/app/jobs/hmis_external_apis/consume_external_form_submissions_job.rb

```diff
@@ -6,6 +6,7 @@
 
 # retrieve and process external form submissions
 class HmisExternalApis::ConsumeExternalFormSubmissionsJob
```

Comment:
@gigxz should this inherit from BaseJob

Reply:
yeah it probably should. looks like we directly invoke it in the hourly task, so I don't think it would really make a difference, right?
https://github.com/greenriver/hmis-warehouse/blob/045cbf9398249621cf8cc0212ae7a9adda5a0b3c/lib/tasks/grda_warehouse.rake#L329

Reply:
Maybe some of these things shouldn't be jobs, just classes with a `run` method on them.  No harm putting them here, but I'd like to think they _could_ be run as jobs if they are in the jobs directory.